pkg/operator/ceph/disruption/clusterdisruption/add.go

```diff
@@ -72,12 +72,6 @@ func Add(mgr manager.Manager, context *controllerconfig.Context) error {
 		}),
 	}
 
-	// Watch for CephClusters
```

Comment:
At each reconcile, do we always look up the latest CephCluster settings for PDBs in case they changed since the last reconcile?

Reply:
The reconciler does get the latest cluster CR instance on every reconcile. So we would get the latest PDB settings on every reconcile.  https://github.com/rook/rook/blob/54560250e92f8f7ee9fc774dfd410463119edc07/pkg/operator/ceph/disruption/clusterdisruption/reconcile.go#L90